setup.cfg

```diff
@@ -47,6 +47,7 @@ install_requires =
     numpy~=1.15
     pybind11~=2.4
     pytest~=5.2
+    tabulate~=0.8.7
```

Comment:
Do you think is it needed to stick to `tabulate==0.8.*` or could we relax the restrictiion to `tabulate~=0.8` ?

Reply:
0.8 should be enough